empire/pkg/runner/runner.go

```diff
@@ -0,0 +1,178 @@
+// package runner provides a simple interface for running docker containers.
```

Comment:
Is this all docker containers?  Only docker containers for specific commands?  I'm sure I'll figure it out from the code, but might be worth giving a little more description here anyway.